### PR TITLE
PIM-5238: fix scroll on multiselect for mass edit

### PR DESCRIPTION
**Pull request by juliensnz**
| Q                 | A
| ----------------- | ---
| Specs             | NA
| Behats            | NA
| Blue CI           | ?
| Changelog updated | N
| Review and 2 GTM  | N

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig [hunk 1]
```diff
@@ -67,10 +67,9 @@
                 'datepicker',
                 'oro/mediator',
                 'routing',
-                'jquery.multiselect',
-                'jquery.multiselect.filter'
+                'oro/multiselect-decorator'
             ],
-            function ($, _, Scopable, CurrencyField, datepicker, mediator, Routing) {
+            function ($, _, Scopable, CurrencyField, datepicker, mediator, Routing, MultiselectDecorator) {
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: :+1:

**Comment by jmleroux** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig [hunk 2]
```diff
@@ -163,6 +168,10 @@
                     var $openButton = $('button.pimmultiselect').addClass('btn btn-group');
                     $openButton.append($('<span>', { 'class': 'caret' })).removeAttr('style');
 
+                    $('#container').scroll(function () {
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/views/MassEditAction/configure/edit-common-attributes.html.twig]: `.on('scroll', function () { ... })` please ; )

**Comment by phaseinducer** [comment]
GTM after comment fix

**Comment by jjanvier** [comment]
on master?

**Pull request closed without merging**